Tests/Fixtures/Item.php

```diff
@@ -32,6 +32,11 @@ public function __construct($id = null, $value = null, $groupName = null, $price
         $this->price = $price;
     }
 
+    public function isPrimaryKeyNull(): bool
+    {
+        return false;
```

Comment:
🔧 Even though it might be always `false` now, please add in a proper check to make it save for future use and possible data manipulation from within tests.
```suggestion
        return null === $this->getId();
```